app/controllers/concerns/secured.rb

```diff
@@ -22,8 +22,13 @@ def authorize
     return if performed?
 
     validation_response = Auth0Client.validate_token(token)
+
     error = validation_response.error
-    return unless error
+    unless error
+      # Create user_id instance variable so that it can be compared against userId recieved from client
+      @user_id = validation_response.decoded_token[0]['sub']
```

Comment:
Minor suggestion: Since this is the Auth0 ID, not the primary key of the `users` table in our DB, how about we name the variable `@user_external_id`? (I debated calling it `@user_auth0_id`, but wanted something less specific to the auth provider we're using.)

Reply:
That is a great idea. I have used that nomenclature for the User table column as well.